examples/supply-chain-app/README.md

```diff
@@ -61,4 +61,6 @@ On the terminal, issue the following commands (steps 1 to 6) and then perform th
 7. Locate the `.vscode/template.launch.json` file
 8. Within that file locate the entry named `"Example: Supply Chain App"`
 9. Copy the VSCode debug definition object from 2) to your `.vscode/launch.json` file
-10. At this point the VSCode `Run and Debug` panel on the left should have an option also titled `"Example: Supply Chain App"` which
\ No newline at end of file
+10. At this point the VSCode `Run and Debug` panel on the left should have an option also titled `"Example: Supply Chain App"` which starts the application
```

Comment:
@elenaizaguirre I went through the steps here and then it worked to the point of me getting the prompt, pasing in the JWT, then the get JWS endpoint succeeding, but then the next endpoint call fails. Can you confirm if all the functionality of the web app works on your end? I'm attaching some screenshots, hopefully they help a little.

Reply:
@petermetz I have cleaned up my environment and compiled the code again to check that I was not using old compiled code but always works for me.

![tempsnip](https://user-images.githubusercontent.com/4309559/150775983-4ec9f50e-6250-49fc-bdbf-5d7c4bdcbe59.png)
<img width="1813" alt="Captura" src="https://user-images.githubusercontent.com/4309559/150776475-a914b4da-f64e-4088-aca0-a00ab4cd60d1.PNG">

Reply:
@petermetz 
I have run it on a newly installed virtual machine (Ubuntu 20.04) and it still works for me with all functionality working. I don't know what's happening...

Reply:
@elenaizaguirre I got to the bottom of it and realized that it's my own fault. (Well, I'll blame Chrome but still it was me).
I was visiting the link that the application prints to the terminal which has a host of `0.0.0.0` instead of `localhost` and Chrome is blocking CORS requests on hosts like that apparently (it was working fine if I did it with cURL so I know for a fact that it's Chrome doing it).

And the reason why the JWS request worked and the others didn't is because the JWS request goes through to the same API server that is hosting the reverse proxy+GUI (so no CORS needed) but the API server that has the quorum connector plugin installed is on a different host and the client side does not do URL rewriting on the node host that it gets from the consortium database so it tries to hit the 4xxx ports while the GUI is running on 3xxx and then Chrome shuts that down.

TLDR: LGTM